slices/libxml2-utils.yaml

```diff
@@ -0,0 +1,17 @@
+package: libxml2-utils
+
+essential:
+  - libxml2-utils_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libxml2_libs
+    contents:
+      /usr/bin/xmlcatalog:
```

Comment:
We should also add something to the test for the `xmlcatalog` utility.
This [page](https://infinitesque.net/articles/2004/using%20xmlcatalog.xhtml#id-1220086676) shows some simple test executions that we could mimic.

Reply:
Btw, on the linked page, the section entitled `Testing Catalog Resolution` shows some sample executions that could help in figuring out an integration test scenario.